pulser-core/pulser/sequence/sequence.py

```diff
@@ -611,9 +612,12 @@ def _config_detuning_map(
         # DMM has Global addressing
         self._add_to_schedule(dmm_name, _TimeSlot("target", -1, 0, self._qids))
         # Manually store the channel declaration as a regular call
-        self._calls.append(
-            _Call("config_detuning_map", (dmm_name, dmm_id, detuning_map), {})
-        )
+        if to_store:
+            self._calls.append(
+                _Call(
+                    "config_detuning_map", (dmm_name, dmm_id, detuning_map), {}
```

Comment:
This is quite problematic because _Call is not used only for serialization to the abstract representation. `dmm_name` is not an argument to `config_detuning_map`, so I think this will cause problems (for example, when building a sequence from scratch from the stored calls). 

Also, the fact that nothing goes wrong suggests that we should be more comprehensive in the unit tests.